### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by PavelBal**
Expanded description of the r-packages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,11 +41,16 @@ Validation and automation tools include for example:
...
+- [ospsuite](https://github.com/Open-Systems-Pharmacology/OSPSuite-R) package provides the functionality of loading, manipulating, and simulating the simulations created in PK-Sim® and MoBi®. It also offers extended workflows such as parameter sensitivity or PK-parameter calculation. The package is described in detail in [R documentation](https://docs.open-systems-pharmacology.org/working-with-r/r-introduction).
+- [TLF](https://github.com/Open-Systems-Pharmacology/TLF-Library) package offers a set of functions and methods for creating standardized reporting **T**ables, **L**ists, and **F**igures.
+- [OSPSuite.ReportingEngine](https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine) for automated generating of model reports.
+- [OSPSuite.ParameterIdentification](https://github.com/Open-Systems-Pharmacology/OSPSuite.ParameterIdentification) provides the functionality of performing parameter identification (i.e., fitting the model to observed data) with simulations. The package is currently under development and everyone is encouraged to contribute.
```
Comment (Yuri05) [inline README.md:51]: should we write ALL R packages in small letters (`tlf`, `ospsuite.reportingengine` etc.)?

File: README.md [hunk 2]
```diff
@@ -41,11 +41,16 @@ Validation and automation tools include for example:
...
+- [TLF](https://github.com/Open-Systems-Pharmacology/TLF-Library) package offers a set of functions and methods for creating standardized reporting **T**ables, **L**ists, and **F**igures.
+- [OSPSuite.ReportingEngine](https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine) for automated generating of model reports.
+- [OSPSuite.ParameterIdentification](https://github.com/Open-Systems-Pharmacology/OSPSuite.ParameterIdentification) provides the functionality of performing parameter identification (i.e., fitting the model to observed data) with simulations. The package is currently under development and everyone is encouraged to contribute. 
+
+**OSP Qualification Framework R packages are not included into the main OSP Suite setup and must be installed separately. Installation instructions are provided in the documentation of the tools or on the GitHub download site.**
```
Comment (Yuri05) [inline README.md:53]: OSP Qualification Framework **and** R packages

File: README.md (hunk 1 above)
Comment (PavelBal) [reply README.md:51]: Maybe, they appear in R (e.g. R-Studio) in small letters.

File: README.md [hunk 3]
```diff
@@ -60,7 +65,7 @@ PK-Sim can also import and export *project snapshots* in [JSON format](https://e
 [![Setup status](https://ci.appveyor.com/api/projects/status/1p3m417amhra2gic/branch/develop?svg=true&passingText=Suite-Setup)](https://ci.appveyor.com/project/open-systems-pharmacology-ci/suite/branch/develop)
 [![PK-Sim status](https://ci.appveyor.com/api/projects/status/65aa66s8aj2tcp45/branch/develop?svg=true&passingText=PK-Sim)](https://ci.appveyor.com/project/open-systems-pharmacology-ci/pk-sim/branch/develop)
 [![MoBi status](https://ci.appveyor.com/api/projects/status/qgv5bpwys5snl7mk/branch/develop?svg=true&passingText=MoBi)](https://ci.appveyor.com/project/open-systems-pharmacology-ci/mobi/branch/develop)
-[![R status](https://ci.appveyor.com/api/projects/status/5ug50xlaot1x59jy/branch/develop?svg=true&passingText=R-Toolbox)](https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-r/branch/develop)
+[![R status](https://ci.appveyor.com/api/projects/status/5ug50xlaot1x59jy/branch/develop?svg=true&passingText=R-package)](https://ci.appveyor.com/project/open-systems-pharmacology-ci/ospsuite-r/branch/develop)
```
Comment (Yuri05) [inline README.md:68]: here I don't like `R-package` in the badge description. With the new description above (where we don't distinguish between `ospsuite-R` and other R packages) it's not clear what `R-package` means.

File: README.md [hunk 4]
```diff
@@ -79,7 +84,7 @@ How to install setups for the Open Systems Pharmacology Suite with PK-Sim® and
 #### OSP Suite 
 Windows 7®, Windows 8®, Windows 10®, Windows Server 2008 R2®, Windows Server 2012 R2®, Windows Server 2016®, Windows Server 2019®
 
-#### R-Toolbox
+#### R package
```
Comment (Yuri05) [inline README.md:87]: R package**s**

**Pull request merged**